### PR TITLE
Create controller

### PR DESCRIPTION
**Pull request by Liam-Scott-Russell**
Adds the controller and the display class, so payslips can be visualised
Currently all tax information is blank as the tax calculator has not been implemented yet

### CODE REVIEW

File: TaxDetails.cs [hunk 1]
```diff
@@ -5,11 +5,13 @@ interface ITaxDetails
         double GrossIncome { get; set; }
         double IncomeTax { get; set; }
         double NetIncome { get; set; }
+        double Super { get; set; }
```
[the remark this answers is not in the document]
Comment (Liam-Scott-Russell) [reply TaxDetails.cs]: This is a good point. The super that is calculated here is the amount of super contributed (in dollars), as opposed to the super percentage that each employee has. I will update the naming here to better reflect what the properties actually are.

File: Controller.cs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            var employee = CreateEmployeeManually();
+            var payPeriod = GetPayPeriodManually();
+            return PayslipFactory.MakePayslip(employee, payPeriod);
+        }
+
+        private Employee CreateEmployeeManually()
+        {
+            var employee = new Employee();
+            
+            Display.AlertUser("Please input your name:");
```
[the remark this answers is not in the document]
Comment (Liam-Scott-Russell) [reply Controller.cs]: Just wanting to clarify what you mean by "input all the data"?

File: TaxDetails.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Liam-Scott-Russell) [reply TaxDetails.cs]: Just changed this and committed it to the most recent PR, [here](https://github.com/Liam-Scott-Russell/Kata-Payslip-Round-2/pull/4/commits/77090ae7fce2346e36b79c964d326abaaf34a2f2) is the link

File: Controller.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (peterchannonmyob) [reply Controller.cs]: Being able to input all the input values, on new lines and then reading all them once you have given them all (Nice to have)

**Sudo Logic**

```
Display Firstname -> input value -> enter ->
display lastname - input value -> enter ->
Once all done, read all the values
```

[the remark this answers is not in the document]
Comment (Liam-Scott-Russell) [reply Controller.cs]: So you want me to read in all of the employee information, and then construct the employee object all at once after I have read the data?

The process would be something like this:
1) Read all data into local variables
2) Create the employee object as follows?
```
employee = new Employee() {
firstName = userSuppliedFirstName,
lastName = userSuppliedLastName,
...
}
```

[the remark this answers is not in the document]
Comment (peterchannonmyob) [reply Controller.cs]: yes, but if you have time to complete this

**Pull request merged**